ScnSoft-Education/LeetCodeTraining/MedianOfTwoSortedArrays6/MedianOfTwoSortedArrays.cs

```diff
@@ -0,0 +1,44 @@
+﻿namespace LeetCodeTraining.MedianOfTwoSortedArrays6
+{
+    public class Solution
+    {
+        public double FindMedianSortedArrays(int[] nums1, int[] nums2)
+        {
+            int m = nums1.Length;
+            int n = nums2.Length;
+            int[] merged = new int[m + n];
+
+            int i = 0, j = 0, k = 0;
```

Comment:
use real name for variables instead magic names

Reply:
fixed variable names